components/brave_ads/core/internal/common/unittest/unittest_profile_pref_registry.cc

```diff
@@ -32,6 +32,8 @@ void RegisterProfilePrefs() {
   RegisterProfileBooleanPref(prefs::kOptedInToNotificationAds, true);
   RegisterProfileInt64Pref(prefs::kMaximumNotificationAdsPerHour, -1);
 
+  RegisterProfileBooleanPref(prefs::kOptedInToSearchAds, true);
```

Comment:
nit: for consistency with rest of brave_ads/core` could we please use `kOptedInToSearchResultAds`